x-pack/test/api_integration/apis/metrics_ui/sources.ts

```diff
@@ -22,10 +22,14 @@ export default function ({ getService }: FtrProviderContext) {
   const kibanaServer = getService('kibanaServer');
 
   describe('sources', () => {
-    before(() => esArchiver.load('x-pack/test/functional/es_archives/infra/metrics_and_logs'));
-    after(() => esArchiver.unload('x-pack/test/functional/es_archives/infra/metrics_and_logs'));
-    before(() => kibanaServer.savedObjects.cleanStandardList());
-    after(() => kibanaServer.savedObjects.cleanStandardList());
+    before(async () => {
+      await esArchiver.load('x-pack/test/functional/es_archives/infra/metrics_and_logs');
+      await kibanaServer.savedObjects.cleanStandardList();
+    });
+    after(async () => {
+      await esArchiver.unload('x-pack/test/functional/es_archives/infra/metrics_and_logs');
+      await kibanaServer.savedObjects.cleanStandardList();
+    });
```

Comment:
`error  Unexpected use of duplicate Mocha `before` hook  mocha/no-sibling-hooks`

Duplicating hooks is not recommended because it can lead to confusion about the order of execution and potential issues in test maintenance.